module3/projects/consultancy/evaluation.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: Consultancy - Evaluation
+type: project
+---
+
+_[Back to Consultancy Home](./index)_ 
+
+## Rubric
```

Comment:
Do we plan to add "Approaching" and "Below" expectation categories? Or are they required to complete all the bullet points of Meets Expectations to pass making the other two categories irrelevant?

Reply:
We actually currently don't have those sections for consultancy nor the FE stretch tech project. It's a bit of a departure from the other projects, but I just kept it the same. I'm not opposed to adding them

Reply:
Especially since they can wiggle their MVP for the first 2 iterations, I think it's not unreasonable to require all the categories to be a passing project. I'd love to hear more about how the FE grades stretch tech. 

Right now with BE, we grade consultancy projects as if all 4 categories were present and the approaching/below grades are sort of assumed.. which is not great I suppose